tap-snapshots/test/lib/commands/publish.js.test.cjs

```diff
@@ -51,7 +51,7 @@ Array [
   ],
   Array [
     "",
-    "Publishing to https://registry.npmjs.org/ (dry-run)",
+    "Publishing to https://registry.npmjs.org/ with tag latest (dry-run)",
```

Comment:
can the tag names be quoted or backticked?

Reply:
we can update that in a subsequent PR